src/main/java/org/skriptlang/skriptworldguard/worldguard/RegionUtils.java

```diff
@@ -136,9 +137,38 @@ public static RegionContainer getRegionContainer() {
 	 * - region data for the given world has failed to load
 	 * - support for regions has been disabled
 	 */
-	@Nullable
-	public static RegionManager getRegionManager(World world) {
+	public static @Nullable RegionManager getRegionManager(World world) {
 		return WorldGuard.getInstance().getPlatform().getRegionContainer().get(BukkitAdapter.adapt(world));
 	}
 
+	/**
+	 * @return All regions of all worlds
+	 */
+	public static WorldGuardRegion @Nullable [] getRegions() {
+		return getRegions(null);
+	}
+
+	/**
+	 * Get all regions of all worlds or provide {@code world} to get all regions of that {@link World}
+	 * @param world The {@link World} to get regions from, or {@code null} to get regions from all worlds
+	 * @return All regions from the world, or all worlds if {@code world} is null.
+	 */
+	public static WorldGuardRegion @Nullable [] getRegions(@Nullable World world) {

```

Comment:
Why are these methods nullable? They don't appear to return null (nor should they - just return an empty array)